test/test.sh

```diff
@@ -68,4 +69,12 @@ export CUSTOM_INIT_REGISTRY_NAMESPACE="$REGISTRY_NAMESPACE"
 export CUSTOM_INIT_NAME_TAG="$INIT_IMAGE_NAME_TAG"
 export IMAGE_PULL_SECRET_NAME="$IMAGE_PULL_SECRET"
 
-make -k all-test FOCUS="$FOCUS" ARGS="$ARGS"
+if [ "$TEST_TYPE" = "cluster-test" ]; then
+   make cluster-test FOCUS="$FOCUS" ARGS="$ARGS"
+elif [ "$TEST_TYPE" = "backup-test" ]; then
+   make backup-service-test FOCUS="$FOCUS" ARGS="$ARGS"
+   make backup-test FOCUS="$FOCUS" ARGS="$ARGS"
+   make restore-test FOCUS="$FOCUS" ARGS="$ARGS"
```

Comment:
Not urgent (can be fixed in upcoming PR): As we are not passing `--focus` in the makefile for other test-suites like backup and restore, its better to remove the FOCUS args from here also. 
@jwalantmodi05 take a note of this.